locales/en/plugin__kubevirt-plugin.json

```diff
@@ -724,6 +725,7 @@
   "No optimized StorageProfile settings for this StorageClass.": "No optimized StorageProfile settings for this StorageClass.",
   "No owner": "No owner",
   "No parameters found in this template.": "No parameters found in this template.",
+  "No Preferences found": "No Preferences found",
```

Comment:
=> "No preferences found"

Reply:
@DanaOrr @apinnick If we look at OCP pages, they're all using the uppercase, such as "No Pods found", "No StatefulSets found", "No DeploymentConfigs found", should we just align with OCP?

Reply:
"No StatefulSets" and "No DeploymentConfigs" are understandable, but not correct, because StatefulSet and DeploymentConfig are k8s objects and therefore are capitalized. However, the writing guidelines say that we are not to use plural forms of k8s objects -- "StatefulSet" is OK but "StatefulSets" is wrong. 

In the case of "preferences" or "instance types", those terms should not be capitalized because they are [general references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references), not k8s object names.